assets/src/components/detours/detourRouteSelectionPanel.tsx

```diff
@@ -63,27 +67,43 @@ export const DetourRouteSelectionPanel = ({
       <Panel.Body className="d-flex flex-column">
         <Panel.Body.ScrollArea className="d-flex flex-column">
           <section className="pb-3">
-            <h2 className="c-diversion-panel__h2" id={selectId}>
-              Choose route
-            </h2>
-            <FormSelect
-              aria-labelledby={selectId}
-              value={selectedRouteInfo.selectedRoute?.id}
-              onChange={(changeEvent) => {
-                onSelectRoute?.(
-                  allRoutes.find(
-                    (route) => route.id === changeEvent.target.value
-                  )
-                )
-              }}
+            <Form
```

Comment:
note: Since we're not actually doing "form" things, and just using it for it's css classes, I think we maybe should use the `as` prop to change this from a `form` to something else, I guess a `div`?

I'd keep it a form if we wrapped all of the contents in the form (RPC and the "submit" button), but those elements aren't really setup to be part of a form either IMO.
```suggestion
            <Form
              as="div"
```